app/controllers/users/invitations_controller.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+class Users::InvitationsController < ApplicationController
+  before_action :set_invitation, only: [:edit, :update, :destroy]
+  skip_before_action :authenticate_user!, only: [:accept]
+  skip_before_action :validate_company!, only: [:accept]
+  skip_after_action :verify_authorized, only: [:accept]
+
+  def create
+    authorize :invitation
+    @invitation = Invitation.new(invitation_params)
+    @invitation.company = current_company
+    @invitation.sender = current_user
+
+    if @invitation.save
+      flash[:success] = t(".success")
```

Comment:
```suggestion
      redirect_to team_index_path, success:  t(".success")
```
Change in other places too.

Reply:
https://github.com/saeloun/miru-web/blob/5c20577e08e2ca99a2946c479e5672aa3b4894a2/app/controllers/users/invitations_controller.rb#L15-L21

We are using a custom method to set error flash messages and redirect users to the same path.

```
 if @invitation.save 
    redirect_to team_index_path, success:  t(".success")
 else 
   set_error_flash 
   redirect_to team_index_path
 end
```

Doing this will be kind of redundant.